.ko.yaml

```diff
@@ -45,3 +45,299 @@ baseImageOverrides:
   k8s.io/test-infra/gcsweb/cmd/gcsweb: gcr.io/k8s-prow/alpine:v20200713-e9b3d9d
   # prow integration test
   k8s.io/test-infra/prow/test/integration/fakeghserver: gcr.io/k8s-prow/alpine:v20200713-e9b3d9d
+
+builds:
```

Comment:
This list must be kept in sync with https://github.com/kubernetes/test-infra/blob/master/prow/.prow-images.yaml, right? We might want to add a presubmit check for this or at least some comments.

Reply:
not necessarily imho, not all prow binaries import `prow/version` package so this setting has not effect on them, and not having a version is not the end of the world for new prow binary either. I think it might be better adding a README somewhere explaining how these work